python/activator/make_pgpass.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
```

Comment:
I could (though it's slightly messier in Python). But I wasn't sure if that would have any security implications like leaving the passwords in the Python interpreter's memory.

Reply:
I believe the environment variables are carried into the Python process (otherwise you won't be able to do this in activator startup).  As a result, anything that can get access to process memory already has them.